crates/kitsune_p2p/kitsune_p2p/src/lib.rs

```diff
@@ -43,8 +43,15 @@ pub mod dependencies {
 }
 
 /// This value determines protocol compatibility.
-/// Any time there is a protocol breaking change, this number must be incremented.
-pub const KITSUNE_PROTOCOL_VERSION: u16 = 0;
+/// Any time there is a protocol breaking change,
+/// this number must be incremented.
+///
+/// Version bump history:
+/// - 0 - 2024-03-11
+///     - The initial protocol version when added to the tx5 preflight.
+/// - 1 - 2024-06-06
+///     - Bumped for the switch from tx5-signal-srv to sbd-server.
+pub const KITSUNE_PROTOCOL_VERSION: u16 = 1;
```

Comment:
@maackle - does this seem appropriate?

Reply:
might be worth a CHANGELOG entry

Reply:
done: https://github.com/holochain/holochain/pull/3984/commits/2f5b9af1245d03d9c513aff6d1dcaa0268f2b125